### ISSUE TITLE
Allow possibility of sending empty text when using editable textarea in trainer page

### ISSUE THREAD

**Issue by mavv-coder** [Lemoncode/code-paster#106]
Right now, trainer can edit old text and send new text using the editable textarea in the bottom. However, sending the editable textarea content empty is not possible. What if we want to start again removing all text? We want to implemente this possibility with a dialog confirm element to warn the user that all text is going to be removed if he clicks "yes". Of course, the confirm element should not be displayed any time we edit the text, but only when we try to send all the content empty.

**Comment by MiguelAngelRL** [comment]
From my point of view a "Reset" button would be more convenient to perform that feature.

**Comment by mavv-coder** [comment]
Good point, @MiguelAngelRL. I like that. However, your option would still need a confirmation from the user, right? Or do you have another idea to avoid the necessity of implementing a dialog element?

**Comment by MiguelAngelRL** [comment]
Absolutely @mavv-coder, a Dialog/Alert component is necessary in any case.

**Comment by mavv-coder** [comment]
Great. Let's wait and see what @brauliodiez thinks about this. I see eye to eye with you on the button solution.
